packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -101,16 +101,18 @@ export interface TableProps {
   ttlAttributeName?: string;
 
   /**
-   * Partition key attribute definition. This is eventually required, but you
-   * can also use `addPartitionKey` to specify the partition key at a later stage.
+   * Table sort key attribute definition.
+   *
+   * @default no sort key
    */
-  partitionKey?: Attribute;
+  sortKey?: Attribute;
+}
 
+export interface TableProps extends TableOptions {
```

Comment:
Why the `TableProps` versus `TableOptions` distinction now?

Reply:
Good catch. leftovers from previous life. merged.